samples/snippets/src/main/java/pubsublite/SubscriberExample.java

```diff
@@ -71,6 +71,12 @@ public static void subscriberExample(
         (PubsubMessage message, AckReplyConsumer consumer) -> {
           System.out.println("Id : " + MessageMetadata.decode(message.getMessageId()));
           System.out.println("Data : " + message.getData().toStringUtf8());
+          System.out.println("Ordering key : " + message.getOrderingKey());
+          System.out.println("Attributes : ");
+          message
```

Comment:
I'd also demonstrate extracting the event time

MessageTransforms. decodeAttributeEventTime(message.getAttributesMap().get(MessageTransforms.PUBSUB_LITE_EVENT_TIME_TIMESTAMP_PROTO))

Reply:
This is a generic SubscriberExample for messages with and without attributes.

Reply:
Ah okay. Thats fine then. Could you add:

```
if (message.getAttributesMap().containsKey(MessageTransforms.PUBSUB_LITE_EVENT_TIME_TIMESTAMP_PROTO)) {
  System.out.println("Event timestamp : ", + MessageTransforms. decodeAttributeEventTime(message.getAttributesMap().get(MessageTransforms.PUBSUB_LITE_EVENT_TIME_TIMESTAMP_PROTO)));
}
```

Reply:
Yep, a `if..else` exists with in the last commit.